src/app/containers/README.md

```diff
@@ -1,36 +0,0 @@
-# Containers
-
-## Overview
-
-This repo follows the container component pattern.
-
-NB containers are just React components
-
-"A container does the data fetching and then renders its corresponding sub-component. That's it." [Medium Blog Post Source](https://medium.com/@learnreact/container-components-c0e67432e005)
-
-Here is a helper video to explain this pattern from Facebook: [Ext. YouTube Link](https://www.youtube.com/watch?v=KYzlpRvWZ6c)
-
-Please refer to [Simorgh's coding standards](https://github.com/bbc/simorgh/blob/latest/docs/Code-Standards.md) documentation for further guidance.
-
-## We strongly recommend using the following webpack plugins
-
-These will reduce the size of your bundle considerably.
-
-```js
-/*
-* The webpack.ContextReplacementPlugin allows us to load only
-* the specific locales we require into our code. By using this
-* plugin we can reference `moment/locale/en.js` in the files
-* we need it, without importing it in every file or importing
-* every locale.
-*/
-new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en-gb/),
```

Comment:
We should retain the information regarding webpack plugin for Moment next to the timestamp containers. But it should be rewritten to be useful for consumers of containers and contributors to Simorgh. Happy for this to be in a separate PR.

Reply:
I don't know how to write the issue to do this as I don't see how it's helpful. Aren't we just saying that once it's in psammead we should make clear that we recommend using this plugin? Doing it before then doesn't make any sense to me, simorgh has no consumers.

Reply:
Sure, we can add it to the relevant container once we move it into Psammead. It lives in git history if someone needs to refer to it at that point.